ooyala/html5-skin
# Feature/player 2945

This PR introduces is an improvement for track change flow. 
It introduces more clear approach to changing active tracks by adding new event - `MULTI_AUDIO_CHANGED`.

### Addresses
[PLAYER-3197](https://jira.corp.ooyala.com:8443/projects/PLAYER/issues/PLAYER-3197)
[PLAYER-3191](https://jira.corp.ooyala.com:8443/projects/PLAYER/issues/PLAYER-3191)
[PLAYER-2945](https://jira.corp.ooyala.com:8443/browse/PLAYER-2945)
[PLAYER-3123](https://jira.corp.ooyala.com:8443/browse/PLAYER-3123)

### Related PRs

[#306](https://github.com/ooyala/html5-video-plugins/pull/306)
[#303](https://git.corp.ooyala.com/projects/PBW/repos/video-plugins-internal/pull-requests/303/overview)

### Summary of changes
ClosedCaptionsMultiAudioMenu
- notify controller about selected audio track instead of id
Html5Skin controller
- subscribe to MULTI_AUDIO_CHANGE
- publish event to notify VideoController about readiness of skinConfig
- publish event to notify VideoController about new selected audio track

### Tests
ClosedCaptionsMultiAudioMenu
- verify that audio track was sent instead of id

Html5Skin controller
- verify that calling setCurrentAudio sends track instead if id
- verify that checking new selected audio track updates previous tracks

- [x] All Unit tests passed without errors

### Misc
Codestyle changes
